pkg/provisioners/helmapplications/virtualcluster/provisioner.go

```diff
@@ -44,6 +45,10 @@ var (
 	})
 )
 
+var (
+	errNoVKCInContext = errors.New("no VirtualKubernetesCluster in context")
```

Comment:
Technically, this is a type assertion error rather than it being totally missing... 😄  Hmm, I wonder why this isn't error checked like everything else.  I'm in core are the moment, may even add in some generic magic to do the assert for you.